### PR TITLE
LP Interop configs update ocp ver to 4.16

### PR DESCRIPTION
**Pull request by smatula**

### CODE REVIEW

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-openshift-grafana-tempo-operator-main-tempo-product-ocp-4.16-lp-interop-tempo-interop-aws periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.16-lp-interop-mtc-interop-aws periodic-ci-rh-openjdk-jdkContainerOcpTests-main-openjdk-ocp4.16-lp-interop-openjdk-interop-aws periodic-ci-redhat-developer-service-binding-operator-release-v1.4.x-4.16-acceptance-lp-interop periodic-ci-openshift-jenkins-release-4.16-jenkins-ocp4.16-lp-interop-jenkins-interop-aws

File: ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.16-lp-interop.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: mtc-interop-aws
+  cron: 0 6 11 2 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      CLUSTER1_CONFIG: name=mtc-aws-ipi-source;base-domain=cspilp.interop.ccitredhat.com;platform=aws;region=us-east-2;version=4.14;stream=stable;worker-flavor=m5.xlarge;timeout=2h
```
Comment (calebevans) [inline ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.16-lp-interop.yaml:57]:
```suggestion
      CLUSTER1_CONFIG: name=mtc-aws-ipi-source;base-domain=cspilp.interop.ccitredhat.com;platform=aws;region=us-east-2;version=4.15;stream=stable;worker-flavor=m5.xlarge;timeout=2h
```

Like you said in your Slack message, this should be 4.15 now

**Review by calebevans — CHANGES_REQUESTED** [review]
Just that MTC change, otherwise lgtm

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-openshift-knative-serverless-operator-main-ocp4.16-lp-interop-operator-e2e-interop-aws-ocp416 periodic-ci-openshift-jenkins-release-4.16-jenkins-ocp4.16-lp-interop-jenkins-interop-aws periodic-ci-quay-quay-tests-master-ocp-416-quay-quay-e2e-tests-quay310-ocp416-lp-interop periodic-ci-sclorg-ansible-tests-master-dotnet-ocp4.16-lp-interop-dotnet-interop-aws periodic-ci-redhat-developer-odo-main-odo-ocp4.16-lp-interop-odo-scenario-aws

**Review by calebevans — APPROVED** [review]
/lgtm

File: ci-operator/jobs/windup/windup-ui-tests/windup-windup-ui-tests-v1.2-periodics.yaml [hunk 2]
```diff
@@ -183,3 +183,84 @@ periodics:
...
+    org: windup
+    repo: windup-ui-tests
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
+    ci-operator.openshift.io/variant: mtr-ocp4.16-lp-interop
+    ci.openshift.io/generator: prowgen
+    job-release: "4.16"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-windup-windup-ui-tests-v1.2-mtr-ocp4.16-lp-interop-mtr-interop-aws
```
Comment (calebevans) [inline ci-operator/jobs/windup/windup-ui-tests/windup-windup-ui-tests-v1.2-periodics.yaml:203]: Just need a slack config

**Review by calebevans — APPROVED** [review]
/lgtm

**Review by IshwarKanse — APPROVED** [review]

**Pull request merged**